modules/etl/camunda/element-templates/liferay-webhook-connector-boundary.json

```diff
@@ -0,0 +1,299 @@
+			"feel": "required",
+			"group": "authorization",
+			"id": "inbound.auth.apiKeyLocator",
+			"label": "Secret Locator",
+			"optional": false,
+			"type": "String",
+			"value": "=get value(request.headers, \"x-api-key\")"
+		},
+		{
+			"binding": {
+				"name": "inbound.responseExpression",
+				"type": "zeebe:property"
+			},
+			"description": "Expression used to generate the HTTP response.",
+			"feel": "required",
+			"group": "webhookResponse",
+			"id": "inbound.responseExpression",
+			"label": "Response Expression",
+			"optional": true,
+			"type": "Text"
+		},
+		{
+			"binding": {
+				"name": "activationCondition",
+				"type": "zeebe:property"
+			},
+			"description": "Condition under which the Connector triggers. Leave empty to catch all events.",
+			"feel": "required",
+			"group": "activation",
+			"id": "activationCondition",
+			"label": "Activation Condition",
+			"optional": true,
+			"type": "String"
+		},
+		{
+			"binding": {
+				"name": "consumeUnmatchedEvents",
+				"type": "zeebe:property"
+			},
+			"group": "activation",
+			"id": "consumeUnmatchedEvents",
+			"label": "Consume Unmatched Events",
+			"tooltip": "Unmatched events are rejected by default, allowing the upstream service to handle the error. Check this box to consume unmatched events and return a success response",
+			"type": "Boolean",
+			"value": true
+		},
+		{
+			"binding": {
+				"name": "correlationKey",
+				"type": "bpmn:Message#zeebe:subscription#property"
+			},
+			"constraints": {
+				"notEmpty": true
+			},
+			"description": "Sets up the correlation key from process variables.",
+			"feel": "required",
+			"group": "correlation",
+			"id": "correlationKeyProcess",
+			"label": "Correlation Key (Process)",
+			"type": "String"
+		},
+		{
+			"binding": {
+				"name": "correlationKeyExpression",
+				"type": "zeebe:property"
+			},
+			"constraints": {
+				"notEmpty": true
+			},
+			"description": "Extracts the correlation key from the incoming message payload.",
+			"feel": "required",
+			"group": "correlation",
+			"id": "correlationKeyPayload",
+			"label": "Correlation Key (Payload)",
+			"type": "String"
+		},
+		{
+			"binding": {
+				"name": "messageIdExpression",
+				"type": "zeebe:property"
+			},
+			"description": "Expression to extract unique identifier of a message.",
+			"feel": "required",
+			"group": "correlation",
+			"id": "messageIdExpression",
+			"label": "Message ID Expression",
+			"optional": true,
+			"type": "String"
+		},
+		{
+			"binding": {
+				"name": "messageTtl",
+				"type": "zeebe:property"
+			},
+			"constraints": {
+				"notEmpty": false,
+				"pattern": {
+					"message": "must be an ISO-8601 duration",
+					"value": "^(PT.*|)$"
+				}
+			},
+			"description": "Time-to-live for the message in the broker (ISO-8601 duration).",
+			"feel": "optional",
+			"group": "correlation",
+			"id": "messageTtl",
+			"label": "Message TTL",
```

Comment:
I don't know about this label because before now I was not familiar with TTL. So I do not know how widely known it is. Maybe

> Message Lifespan (Time-To-Live)